docs/UserGuide.md

```diff
@@ -259,28 +281,35 @@ our definition of identical internships further, Flagship will not be able to ca
-ℹ️ We allow you to create internship entries with application deadlines and start dates before today's current date. It is designed as such 
-so that you can track your past internship applications for your own reference. 
-</div>
 
 <br>
 
+
 ### Editing an Internship: `modify`
 
-Update details of an existing internship entry.
+Need to update an existing internship entry? The `modify` command lets you adjust the details of your applications in Flagship. Whether the application deadline has changed or you've picked up a new skill that meets the job requirements, ensure your internship details are current.
 
-**Format**: `modify INDEX c/COMPANY_NAME ro/ROLE a/APPLICATION_STATUS s/START_DATE de/DEADLINE du/DURATION re/REQUIREMENTS...`
+**Format**: `modify INDEX c/COMPANY_NAME ro/ROLE a/APPLICATION_STATUS de/DEADLINE s/START_DATE du/DURATION [re/REQUIREMENTS]...`
 
-* Modify the internship at the specified `INDEX`. This index correlates with the position in the displayed internship list. It **must be a positive integer**, e.g., 1, 2, 3, …​.
-* New inputs will overwrite existing values.
-* Not all categories need to be specified.
-* Categories that are not included will keep the previous values.
-* Editing requirements will replace current requirements; adding requirements doesn't accumulate.
-* Use the command without `re/` to remove all of an internship's requirements.
+**Example**: `modify 1 c/Jane Street ro/Coffee maker a/Yet to apply de/29/11/2022 s/20/01/2023 du/3 re/C++ re/Coffee`
 
-**Examples**:
-* `modify 1 c/Jane Street ro/Coffee maker a/Yet to apply s/20/01/2023 de/29/11/2022 du/3 re/C++ re/Coffee`
+**Interpretation**: Update the first internship entry on your Flagship list. The role is at **Jane Street** for a **Coffee maker** position. You have **yet to apply**. The **application deadline** is set for 29 November 2022, and the start date is 20 January 2023. The internship is for a **3-month** period, with requirements for proficiency in **C++** and experience in **Coffee making**.
+
+<div markdown="block" class="alert alert-info">
+💡 You can selectively update details for an internship entry. Attributes not included in the command will maintain their existing values.
+</div>
+
+<div markdown="span" class="alert alert-warning">
+
+⚠️ **Internship Entry Modification Policies:**
+
+- **INDEX is Mandatory**: The INDEX, a positive integer, is required to identify the entry's position in the list. Modifying an entry will replace the existing list, not add to it.
+- **No Duplicate Company Name and Role**: Editing to create a duplicate entry with the same company name and role as an existing internship entry is not permitted, ensuring consistency with the creation section's prohibition of duplicates.
+- **Parameter Requirement**: There must be at least one parameter following the INDEX when modifying an entry. Omission of parameters will trigger a warning.
+</div>
+
+**Expected Output**:
+![Sort](images/Modify.png)
 
```

Comment:
I dont think u labelled this correctly. It should be modify

Reply:
added back